packages/ag-charts-community/src/chart/interaction/zoomManager.ts

```diff
@@ -238,19 +238,31 @@ export class ZoomManager extends BaseManager<ZoomEvents['type'], ZoomEvents> imp
     }
 
     public updateZoom(callerId: string, newZoom?: AxisZoomState) {
-        if (this.axisZoomManagers.size === 0) {
-            const stateId = this.state.stateId()!;
+        const { axisZoomManagers, state } = this;
+
+        if (axisZoomManagers.size === 0) {
+            const stateId = state.stateId()!;
             if (stateId === 'initial' || stateId === callerId) {
-                this.state.set(callerId, newZoom);
+                state.set(callerId, newZoom);
             }
             return;
         }
 
-        this.state.set(callerId, newZoom);
+        state.set(callerId, newZoom);
 
-        this.axisZoomManagers.forEach((axis) => {
-            axis.updateZoom(callerId, newZoom?.[axis.getDirection()]);
-        });
+        if (this.independentAxes) {
```

Comment:
I don't think this check belongs here as the `updateZoom()` fn is expected to update all axes.